notebooks/official/prediction/pytorch_train_deploy_models_with_prebuilt_containers.ipynb

```diff
@@ -0,0 +1,1400 @@
+{
```

Comment:
<p>Consider condensing or removing all of this. A lot of the info was already covered in the intro and presumably there are other codelabs for custom container.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1728/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>